example/lib/pages/prejoin.dart

```diff
@@ -177,24 +177,19 @@ class _PreJoinPageState extends State<PreJoinPage> {
         await keyProvider.setKey(args.e2eeKey!);
       }
 
-      // Try to connect to the room
-      // This will throw an Exception if it fails for any reason.
-      await room.connect(
-        args.url,
-        args.token,
+      final room = Room(
         roomOptions: RoomOptions(
           adaptiveStream: args.adaptiveStream,
           dynacast: args.dynacast,
           defaultAudioPublishOptions: const AudioPublishOptions(
             name: 'custom_audio_track_name',
           ),
-          defaultVideoPublishOptions: VideoPublishOptions(
-            simulcast: args.simulcast,
-            videoCodec: args.preferredCodec,
-            backupVideoCodec: BackupVideoCodec(
-              enabled: args.enableBackupVideoCodec,
-            ),
-          ),
+          defaultCameraCaptureOptions: CameraCaptureOptions(
+              maxFrameRate: 30,
+              params: VideoParameters(
+                dimensions: const VideoDimensions(1280, 720),
+                encoding: encoding,
```

Comment:
is there a way to not set this in example? I think it would be confusing to users why they'd have to set this